### PR TITLE
Updated oldDataWarning function to display how many months old caniuse-lite is.

### PR DESCRIPTION
**Pull request by mezhnin**

### CODE REVIEW

File: node.js [hunk 1]
```diff
@@ -419,8 +429,10 @@ module.exports = {
     var halfYearAgo = Date.now() - TIME_TO_UPDATE_CANIUSE
 
     if (latest !== 0 && latest < halfYearAgo) {
+      var monthPassed = getMonthsPassed(latest)
+
       console.warn(
-        'Browserslist: caniuse-lite is outdated. Please run:\n' +
+        'Browserslist: caniuse-lite is '+ monthPassed +' month old. Please run:\n' +
```
Comment (ai) [inline node.js:435]: We need plural form `month`/`months` and tests

Comment (mezhnin) [reply node.js:435]: ok, let me fix it

File: node.js [hunk 2]
```diff
@@ -419,8 +429,10 @@ module.exports = {
     var halfYearAgo = Date.now() - TIME_TO_UPDATE_CANIUSE
```
Comment (ai) [inline node.js:429]: Why you need `halfYearAgo` if you already calculated month?

File: node.js (hunk 1 above)
Comment (mezhnin) [reply node.js]: fixed

File: node.js (hunk 2 above)
Comment (mezhnin) [reply node.js]: fixed

**Pull request merged**